cms/templates/molecules.html

```diff
@@ -0,0 +1,105 @@
+{% import 'atoms.html' as atoms %}
+
+{% macro nav_inline_centered(languages) -%}
+<ul class="nav list-inline l-tac" role="navigation">
+{% for key, value in languages %}
+	<li><a href="/locale/{{key}}/">{{value}}</a></li>
+{% endfor %}
+	<li><a href="/spice/locale/change/">&gt;&gt;</a></li>
+</ul>
+{%- endmacro %}
+
+{% macro banner_with_image(src) -%}
+<div class ="banner">
+	{{ atoms.image(src, "", "")}}
+</div>
+{%- endmacro %}
+
+{% macro image_and_heading(view, cat) -%}
+{{atoms.image( view.get_image_url(cat.image_host, cat.image, 45, 45), "" , "" ) }}
+<h3>{{cat.title}}</h3>
+{%- endmacro %}
+
+
+{% macro nav_inline_highlight_primary() -%}
```

Comment:
Same issue with naming here, I think this should be a more descriptive name. Also, I don't think these blocks need to be hardcoded. Perhaps they should take a list of tuples `(route_name, title)` e.g `site_menu_links([('home', 'Home'), ('about', 'About')])` etc.